foundations/javascript_basics/fundamentals-5.md

```diff
@@ -21,18 +21,18 @@ Besides being a quick and handy way to store data, arrays also have a set of fun
 1. Start out by watching [this video](https://www.youtube.com/watch?v=HB1ZC7czKRs) from Wes Bos.  To follow along, use your local Javascript30 repository. If you don't have the repo yet, clone it from [here](https://github.com/wesbos/JavaScript30).
 2. Watch and code with [Part 2](https://www.youtube.com/watch?v=QNmRfyNg1lw) of Wes Bos's array series.
 
-### Practice
+### Assignment
 
 You have been through a _ton_ of content, and it would not be surprising if you were feeling rather overwhelmed at this point.  Let's take a little time to slow down and practice this stuff with more coding exercises.  I'll warn you now: some of these get pretty difficult, but if you've done the lessons you should have covered everything you need to know to solve them.  Don't be afraid to go back and review if you find that you can't remember something, that's part of the process.  If you get stuck come join us in the chatroom!
```

Comment:
I think this line can also be removed

Reply:
Done. https://github.com/TheOdinProject/curriculum/pull/24248/commits/8e620c89862d505387b7aa7f5c9439a8302cd2f1

Although formatting consistency was prompted here for sure, I wasn't certain if I was ditching an important part or not.